Assignment2/hunter097-1.b.ttl

```diff
@@ -0,0 +1,18 @@
+@base < www.oeg upm.net / person >
+@prefix : < www.oeg upm.net / person #>
+@prefix person : < www.oeg upm.net / person #>
+
+:Class a rdfs:Class.
+	
+:Sensor029 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+:Computer101 a rdfs:Class;
+	rdfs:subClassOf:Class.
+
+Measurement8401:hasMeasurement a rdf:Sensor029;
+	rdfs:hasTemperature:29;
+	rdfs:atTime:2010-06-12T 12:00:12.

```

Comment:
Problem with syntax. This will not parse

Reply:
I think I fixed the error